File: Sort an Array.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var sortArray = function(nums) {
    // Тусгай сорилтыг ашиглан тоонуудыг эрэмбэлнэ
    return nums.sort((a, b) => a - b);  // Тоонуудыг багаас том руу эрэмбэлэх
};

// Жишээ хэрэглээ:
console.log(sortArray([5, 2, 9, 1, 5, 6])); // Гаргах: [1, 2, 5, 5, 6, 9]

File: Sort an Array.py
class Solution:
    def sortArray(self, nums):

        if len(nums) <= 1:
            return nums

        mid = len(nums) // 2
        left_half = self.sortArray(nums[:mid])
        right_half = self.sortArray(nums[mid:])


        return self.merge(left_half, right_half)

    def merge(self, left, right):
        sorted_array = []
        i = j = 0


        while i < len(left) and j < len(right):
            if left[i] < right[j]:
                sorted_array.append(left[i])
                i += 1
            else:
                sorted_array.append(right[j])
                j += 1

        sorted_array.extend(left[i:])
        sorted_array.extend(right[j:])

        return sorted_array
